### DISCUSSION TITLE
Refund for copilot

### DISCUSSION THREAD

**Discussion by SergenKaraoglan** [community/community#45645] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Forgot to cancel before the end of free trial but have not used since. I would appreciate a response from staff.

**Comment by jge162** [comment]
Hello @SergenKaraoglan here is the link to support. Just click on [billing](https://support.github.com/request)  and submit your request. Otherwise I’m sure an admin of discussion will be with you soon. Thank you.

**Comment by AsadullahFarooqi** [comment]
I have kind of a similar issue @jge162 
I wanted to try the free trial as it says "2 months free trial" while registering it asked for the card, I assumed I won't/shouldn't get charged before the free trial. But what happened is that i as soon i submitted i got charged. I would like to my refund. I'll use the free trial if i liked it then I'll subscribe otherwise I'll cancel.

**Comment by Akash1134** [comment, marked as answer]
Hi Everyone, 

We now have the option to self-serve refunds for Copilot. Please head over to the [Help with your GitHub account](https://support.github.com/contact/account?flow=refund_copilot_individual) section and save yourself some time.

Note that, in most cases, the Virtual Agent may be able to offer an immediate refund. But, if your concern is not resolved, you can always use the contact form to get in touch with our billing team.

**Reply by illiarerun** [reply]
@Akash1134 I don't know if it's a bug, but the "Virtual Agent" option is not visible to me 😢 
![image](https://user-images.githubusercontent.com/37161315/235898207-59915cc4-1118-4992-b086-d6253bbd6388.png)

**Reply by zero19124** [reply]
solved my problem! thx

**Comment by zhzhouq9** [comment]
我在6月份很早的时候已经取消订阅了 Copilot 但是在今天还是进行了扣费
